hugo/content/docs/langium-overview.md

```diff
@@ -4,35 +4,35 @@ weight: 0
-This is the point where Langium comes into play. The framework aims to lower the entry point for creating a language by taking care of the technical complexity, allowing you to focus on your domain's requirements.
+This is the point where Langium comes into play. Langium aims to lower the barrier to entry for creating a language by removing the technical complexity, allowing you to focus on your domain's requirements.
 
-In this chapter you'll get a closer look at the requirements programming languages usually have to implement by themselves:
+In this chapter, you'll get a closer look at the requirements developers usually have to implement by themselves when building a programming language:
 
 - [Language parsing](#language-parsing)
 - [Semantic models](#semantic-models)
 - [Cross references and linking](#cross-references-and-linking)
 - [Workspace management](#workspace-management)
 - [Editor support](#editing-support)
 
-With that in mind, Langium provides out-of-the-box solutions for these problems, with the ability to fine-tune every part of it to fit your domain requirements.
+Langium provides out-of-the-box solutions for these problems, with the ability to fine-tune every part of it to fit your domain requirements.
 
 ---
 
 ## Language Parsing
 
-Programming languages and domain specific languages (DSL) cannot be parsed using simple regular expressions (RegExp). Instead they require a more sophisticated parsing strategy. To define a custom language, you'll interact with a high level representation of your context-free grammar using the [Langium grammar language](../grammar-language), in a similar fashion to EBNF.
+Programming languages and domain specific languages (DSLs) cannot be parsed using simple regular expressions (RegExp). Instead they require a more sophisticated parsing strategy. To define a custom language in Langium, you interact with a high level representation of your context-free grammar using the [Langium grammar language](/docs/grammar-language), in a similar fashion to EBNF.
 
-Based on the grammar, Langium is then able to construct a parser which transforms an input string into a semantic model representation.
+Based on the grammar, Langium is then able to construct a parser which transforms an input string into a semantic model representation. Just as the name suggests, this model captures the essential structure to describe your language.
 
 ## Semantic Models
 
-Langium grammars are not only used to parse your language, but also generate a semantic model for your Language as TypeScript interfaces, which automatically match the parsed abstract syntax tree. The following language snippet:
+Langium grammars are not only used to parse your language, but also to generate a semantic model for your Language as TypeScript interfaces. When a program in your language is then parsed, the generated AST will be automatically produced using these interfaces. The following language snippet parses a simple object { name: 'John' }.
```

Comment:
Non-blocking comment. The simple object is now unformatted as code, but it was before, however it's still fine to read.